src/langchain_google_cloud_sql_pg/vectorstore.py

```diff
@@ -461,7 +461,7 @@ async def __query_collection(
         search_function = self.distance_strategy.search_function
 
         filter = f"WHERE {filter}" if filter else ""
-        stmt = f"SELECT *, {search_function}({self.embedding_column}, '{embedding}') as distance FROM {self.table_name} {filter} ORDER BY {self.embedding_column} {operator} '{embedding}' LIMIT {k};"
+        stmt = f"SELECT *, {search_function}({self.embedding_column}, '{embedding}') as distance FROM \"{self.table_name}\" {filter} ORDER BY {self.embedding_column} {operator} '{embedding}' LIMIT {k};"
```

Comment:
nit: for consistency should we follow the escape pattern of the other variables (embedding) in the query and use single quotes? or should they also be using backslashes?
```suggestion
        stmt = f"SELECT *, {search_function}({self.embedding_column}, '{embedding}') as distance FROM '{self.table_name}' {filter} ORDER BY {self.embedding_column} {operator} '{embedding}' LIMIT {k};"
```

Reply:
The table name needs to use double quotes: `<class 'asyncpg.exceptions.PostgresSyntaxError'>: syntax error at or near "'test_tableb5955ff2_5a51_460a_b674_105101e6436f'"` but the embeddings need to be wrapped in a single quote.

Reply:
awesome, good to know